setup.py

```diff
@@ -17,10 +17,9 @@ def read_file(filename):
 
     install_requires=[
         "requests>=1.2.0",
-        "django-tables2==1.21.*",
+        "django-tables2>=2.1.1",
```

Comment:
Older versions of `django-tables2` relied on Python 2 compatibility utilities within Django that have been removed from Django 3.2.